src/App.js

```diff
@@ -4,6 +4,17 @@ import PrivateRoute from "./PrivateRoute";
 import Auth0 from "./Auth0";
 import Home from "./views/Home";
 import OrgTopLevel from "./views/Organization";
+import { ApolloProvider } from '@apollo/react-hooks';
+import ApolloClient from 'apollo-boost';
+import { ApolloTest } from "./ApolloTest";
```

Comment:
does this need to be pulled in here? From what I can see here, it isn't being used in this file

Reply:
Hmm `ApolloProvider` is used on the line 52 and `ApolloProvider` on the line 15. `ApolloTest` isn't used anywhere, I've deleted that one

Reply:
that's the one I meant :)